### PR TITLE
Fix ProfileList component

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #Remove Zindex

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/core/src/A.js [hunk 1]
```diff
@@ -19,4 +19,8 @@ A.propTypes = {
   href: PropTypes.string.isRequired,
 };
 
+A.defaultProps = {
+  href: ''
```
Comment (kilemensi) [inline packages/core/src/A.js]: `href` is required & hence won't need default prop

File: packages/core/src/ScrollableGridList/index.js [hunk 2]
```diff
@@ -117,8 +117,8 @@ ScrollableGridList.propTypes = {
   autoHide: PropTypes.bool,
   cellHeight: PropTypes.number,
   children: PropTypes.node,
-  height: PropTypes.oneOfType(PropTypes.number, PropTypes.string).isRequired, // in px
-  list: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  height: PropTypes.number,
```
Comment (kilemensi) [inline packages/core/src/ScrollableGridList/index.js]: `height` can be `auto`, `2rem`, etc. hence we need to support both `number` & `string` types

File: packages/core/src/ProfileList/useStyles.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ import { makeStyles } from "@material-ui/core/styles";
 
 const useStyles = makeStyles(() => ({
   root: {
-    zIndex: 1,
```
Comment (kilemensi) [inline packages/core/src/ProfileList/useStyles.js:6]: Sure no impact by removing this?

**Review by kilemensi — APPROVED** [review]
:rocket: @nerdlet 

-----------------------

`yarn lint` before merging

File: packages/core/src/ProfileList/useStyles.js (hunk 3 above)
Comment (nerdlet) [reply packages/core/src/ProfileList/useStyles.js:6]: @kilemensi Yap

File: packages/core/src/A.js (hunk 1 above)
Comment (nerdlet) [reply packages/core/src/A.js]: @kilemensi What happens when we don't supply a `href` value?

Comment (kilemensi) [reply packages/core/src/A.js]: You should get an error @nerdlet ... otherwise why use this component if you don't have a link?

Comment (nerdlet) [reply packages/core/src/A.js]: Got it...In that case, I am gonna just leave it as is

**Comment by nerdlet** [comment]
@kilemensi Done

**Pull request merged**